build.gradle

```diff
@@ -651,6 +651,37 @@ task testDocker {
   }
 }
 
+task acceptanceTestsGoQuorum {
+  dependsOn distDocker
+  /**
+   * Tags Description
+   *
+   * Basic tests for private and public tx: basic
+   * Basic tests for Istanbul/IBFT consensus like testing the fault tolerancy: besu-istanbul
+   * Start a Besu/EthSigner/Tessera network with IBFT2: networks/typical-besu::ibft2
+   *
+   * Not available features in Besu: privacy-enhancements-disabled, extension
+   * Not available RPC methods in Besu: async, storage-root, get-quorum-payload, personal-api-signed
+   */
+  def tags = "((basic || basic-istanbul) && !privacy-enhancements-disabled && !async && !extension && !storage-root && !get-quorum-payload && !personal-api-signed) || networks/typical-besu::ibft2"
```

Comment:
FYI @pinges @macfarla those are the tags you should use when running the GoQ AT against Besu